x-pack/test_serverless/functional/test_suites/search/search_index_detail.ts

```diff
@@ -184,11 +184,44 @@ export default function ({ getPageObjects, getService }: FtrProviderContext) {
       before(async () => {
         await es.indices.create({ index: indexName });
         await security.testUser.setRoles(['index_management_user']);
+      });
+      beforeEach(async () => {
         await pageObjects.common.navigateToApp('indexManagement');
         // Navigate to the indices tab
         await pageObjects.indexManagement.changeTabs('indicesTab');
         await pageObjects.header.waitUntilLoadingHasFinished();
       });
+      after(async () => {
+        await esDeleteAllIndices(indexName);
+      });
+      describe('manage index action', () => {
```

Comment:
We should add a similar test to this in `x-pack/test_serverless/functional/test_suites/common/management/index_management/indices.ts` to test when we there is not a registered index details url to ensure we don't break that experience.

Reply:
added in d8c6fd8312800ad1442f51579dd7f8e894a0c8c1